scss/modules/_code-snippets.scss

```diff
@@ -0,0 +1,118 @@
+////
+/// @author       Web Team at Canonical Ltd
+/// @link         http://ubuntudesign.github.io/vanilla-framework/docs/#mixin-vf-code-snippets
+/// @since        0.0.71
+////
+
+/// Code block and code snippets styles
+/// @group Codesnippets
+/// @example
+///   <pre>
+///     <code>...</code>
+///   </pre>
+@mixin vf-code-snippets {
+
+  code,
+  samp,
+  kbd {
+    font-family: 'Ubuntu Mono', 'Consolas', 'Monaco', 'Lucida Console', 'Courier New', Courier, monospace;
+    text-align: left;
+  }
+
+  pre,
+  code {
+    text-align: left;
+    white-space: pre-line;
+    word-spacing: normal;
+    word-wrap: break-word;
+    tab-size: 4;
+    hyphens: none;
+    direction: ltr;
+  }
+
+  pre,
+  .code-block {
+    overflow: auto;
+    margin: .5em 0;
+    padding: 1em;
+    border: 1px solid $mid-grey;
+    border-radius: .5em;
+
+    code,
+    .code-block__line {
+      float: left;
+      clear: both;
+      margin: 0;
+    }
+
+    &.code-block--terminal {
```

Comment:
Is the parent selector really required here (and below)? Using it greatly increased specificity of the selector.